selfdrive/car/toyota/carcontroller.py

```diff
@@ -137,18 +137,22 @@ def update(self, enabled, CS, frame, actuators,
     # steer torque
     apply_steer = int(round(actuators.steer * SteerLimitParams.STEER_MAX))
 
-    apply_steer = apply_toyota_steer_torque_limits(apply_steer, self.last_steer, CS.steer_torque_motor, SteerLimitParams)
-
     # only cut torque when steer state is a known fault
     if CS.steer_state in [9, 25]:
       self.last_fault_frame = frame
 
     # Cut steering for 2s after fault
-    if not enabled or (frame - self.last_fault_frame < 200):
+    if (frame - self.last_fault_frame < 200) or abs(CS.angle_steers) > 100 or abs(CS.angle_steers_rate) > 100:
```

Comment:
do you feel confident about those hardcoded `100`, or you think they are in the right ballpark? 100 deg steer_angle seems reasonably high, while, 100 deg/s angle_steers_rate seems easily achievable in normal operations.